cmd/rest/main.go

```diff
@@ -30,3 +40,21 @@ func Run() error {
 	}
 	return nil
 }
+
+func connectDB(opt cfg.Options) (*sql.DB, error) {
+	str := fmt.Sprintf("host=%v port=%v user=%v password=%v dbname=%v sslmode=disable",
+		opt.DBConfig.Host, opt.DBConfig.Port, opt.DBConfig.Username, opt.DBConfig.password, opt.DBConfig.DBName)
+
+	db, err := sql.Open("postgres", str)
+	if err != nil {
+		return nil, fmt.Errorf("failed to open database connection: %w", err)
+	}
+
+	if err := db.Ping(); err != nil {
+		return nil, fmt.Errorf("ping to database failed: %w", err)
+	}
+
+	log.Println("Connected to DB")
```

Comment:
It's better to move this print to main func.

Reply:
It seemed to me that nothing was needed in the main function except for Run(). Is it possible to run the log in the Run() function? (As result of the successful call of the migrate func)

Reply:
Ohh, you are right. Yeah, we have to log inside the Run func.

Reply:
Done